### PR TITLE
Lucky ducko patch 2

### PR DESCRIPTION
**Pull request by LuckyDucko**
Just would like you to look it over one please

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -9,15 +9,18 @@ on:
...
+    name: Build ${{ matrix.configuration }}
     runs-on: windows-2022
+    strategy:
+      matrix:
+        configuration: ['Debug', 'Release']
     steps:
     - uses: actions/checkout@v2
     - name: Install workload
-      run: dotnet workload install maui
+      run: dotnet workload install maui-android
```
Comment (maxkoshevoi) [inline .github/workflows/build.yml:20]: You need full MAUI workload, since this package also supports iOS

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -9,15 +9,18 @@ on:
 
 jobs: 
   build:
-    name: Build
+    name: Build ${{ matrix.configuration }}
     runs-on: windows-2022
+    strategy:
+      matrix:
+        configuration: ['Debug', 'Release']
```
Comment (maxkoshevoi) [inline .github/workflows/build.yml:16]: I haven't seen any `#if DEBUG` in this package, so just building Release configuration should be fine

**Review by maxkoshevoi — COMMENTED** [review]
Overall `Build` action seems fine as is (changes in this PR are not needed). 
Something else must be causing a build break. I'll try to have a look tomorrow when I get time

**Comment by maxkoshevoi** [comment]
Here's what seems to be the current issue: `Microsoft.Maui.Resizetizer.targets(522,9): error : Unable to find foreground file`

https://github.com/LuckyDucko/Mopups/runs/7149778613?check_suite_focus=true

**Comment by LuckyDucko** [comment]
@maxkoshevoi hmm, resizetizer always seems to have weird issues with this build. 


Also, I am quite..novice at making these CI pipelines to be honest, I'll need some time for a proper sit down to look through them.

**Pull request closed without merging**